src/app/cake/handlers/get.handler.ts

```diff
@@ -0,0 +1,27 @@
+import { RequestHandler, Request, Response } from "express";
+import { getData, getDataById } from '../cake.service';
+
+type Params = {};
+type Query = {};
+type Body = {};
+type Req = Request<Params, {}, Body, Query>;
+type Res = Response;
+
+export const handlerAll: RequestHandler[] = [
+  // auth
+  async (req: Req, res: Res) => {
+    console.log('Handling GET All...');
+    const data = await getData();
+    console.log('Returning data:', data)
+    res.json(data);
+  }
+];
+
+export const handlerById: RequestHandler[] = [
```

Comment:
I recommend creating a new file `get-by-id.handler` to put this handler in his own file.

Reply:
Done ✅